src/miner.erl

```diff
@@ -652,3 +649,8 @@ process_bbas(N, BBAs) ->
                     <<>>
             end
     end.
+
+catchup_time(N) when N < 0.15 ->
```

Comment:
Do we want equality here?

Reply:
I don't think it really matters here. I *am* interested in what 0.15 actually means here though. I thought N was going to be in seconds.

Reply:
N is a float, it's the result of `BlockTime  - (LastBlockTimestamp - StartBlockTime) / (Height - ActualStartHeight)`, which will be a float.  That said, there is a bug here: if the difference is 0, or very very small, we should return 0.

Reply:
Is it possible that `Height - ActualStartHeight` can ever be 0 ?